lib/openapi/index.ts

```diff
@@ -0,0 +1,46 @@
+import { createDocument } from "zod-openapi"
+
+import { machinesPaths } from "./machines"
+import { proofsPaths } from "./proofs"
+
+export const document = createDocument({
+  openapi: "3.1.0",
+  info: {
+    title: "Ethproofs API",
+    description: `This document outlines the available API endpoints for ethproofs.\n\n**Authentication**\n\nAll endpoints require authentication using an API key in the request header:\n\n\`Authorization: Bearer <api_key>\``,
+    version: "0.0.1",
+  },
+  servers: [
+    {
+      url: "https://main--ethproofs.netlify.app/api/v0",
```

Comment:
Similar to above, we have an env var / constant we could potentially use here (`SITE_URL`) to keep things canonical

```suggestion
      url: new URL("api/v0", SITE_URL).toString(),
```

```tsx
import { SITE_URL } from "@/lib/constants"
```

Reply:
Good point. I've added 2 servers for now (prod and preview urls).